### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by sklirg**
Tests:
- Some helper functions
- Database

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -3,18 +3,29 @@ package main
...
+  }
 
 	engine := gin.Default()
 	AddRoutes(version, engine)
 
-	connectToDb()
-
 	loadDefaultTemplates("templates/")
-	engine.Run(":80")
+	engine.Run(":8002")
```
Comment (sklirg) [inline server.go]: wil lfixxxxxxxxx

File: db_test.go [hunk 2]
```diff
@@ -1,19 +1,99 @@
...
+    return db.Ping() == nil
+  })
   
-  if (len(templates) > 1) {
-    // wtf
-  } else {
-    fmt.Printf("%v", templates)
-    assert.Equal(t, "artikkel", "templates[0][\"name\"]")
+  if err != nil {
+    // log.Fatalf("Could not connect to database: %s", err)
```
Comment (paalka) [inline db_test.go]: Uncommenting this, and adding "log" to the list of imports makes this work again :)

**Comment by paalka** [comment]
Hmmm.. Have you run the new files through go fmt? :)

**Comment by sklirg** [comment]
No! :-(

On 13 May 2016 at 20:40, Pål Karlsrud notifications@github.com wrote:

> Hmmm.. Have you run the new files through go fmt? :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> https://github.com/microserv/publishing-templates/pull/7#issuecomment-219126429

## 

Håkon Solbjørg
(+47) 46 63 77 60

**Pull request merged**